parlai/chat_service/README.md

```diff
@@ -0,0 +1,65 @@
+# Setting up ParlAI for a custom chat-service
+This is an instruction manual to be used as reference on how to configure ParlAI for an arbitrary chat service.
+
+## Message Format
+To maintain consistency we are trying to enforce a deterministic message format throughout this task. If a particular chat service doesn't adhere to this format, one must use ```restructure_message()``` to adapt the messages to this format before the messages being used. The message format is defined below:
+```config
+{
+  mid: # ID of this message
+  recipient: {id: #id of message recipient}
+  sender: {id: #id of message sender}
+  text: # text of the message
+  attachment: # attachment of the message
+}
+```
+### TBD
```

Comment:
Maybe here is where you can define "additional message fields" and just briefly discuss these examples from Messenger

Reply:
So @domrigoglioso and I discussed a few of these and I am pushing in my next commit what we thought, please correct us if we are wrong. Still unsure about what messaging_type and persona_id meant so would need your help on that.

Reply:
`messaging_type` is whether the message is a text message or an image upload; `persona_id` is so that you can have multiple "personas" messaging on messenger, e.g. if you're talking to a bot it can be more than one "persona." That's fine to leave until next commit